doc/lua_api.txt

```diff
@@ -1931,7 +1931,10 @@ Helper functions
     * e.g. `string:split("a,b", ",") == {"a","b"}`
 * `string:trim()`
     * e.g. `string.trim("\n \t\tfoo bar\t ") == "foo bar"`
-* `minetest.pos_to_string({x=X,y=Y,z=Z}, decimal_places))`: returns `"(X,Y,Z)"`
+* `minetest.splittext(str, limit)`: returns a string
+    * Adds new lines to the string to keep it within the specified character limit
+    * limit: Maximal amount of characters in one line
```

Comment:
Isn't this just a hack that the main menu uses to work around line wrapping not being available?  I don't think it should really be exposed.
And if so, it should be `split_text`, or rather something more descriptive like `wrap_text`.